src/lib/integration_test_cloud_engine/mina_automation.ml

```diff
@@ -19,6 +19,8 @@ let cluster_region = "us-west1"
 
 let cluster_zone = "us-west1a"
 
+let enable_working_dir_persitence = true;
```

Comment:
We actually don't need this for the integration tests. The integration tests don't need a PVC because we do not restart the pod when we restart the daemon. The integration testing DSL already supports selecting if you want to restart a node from a fresh state or from and existing state. So we don't need to expose this option, and can always set it to "false" for the integration test deploys.

Reply:
We definitely need PVC for any non-integration test deployment. Integration tests are just special in that the mina daemon is not the root process. Instead, there is a puppeteer python script as the root process which keeps the container alive across mina restarts, allowing the integration test framework to instruct the puppeteer how to control the daemon.